### PR TITLE
Add sin function

### PR DESCRIPTION
**Pull request by BenLorantfy**
Added sin function to math.js

### CODE REVIEW

File: math.js [hunk 1]
```diff
@@ -20,4 +20,8 @@ export function sqrt(a) {
 
 export function log(a) {
     return Math.log(a);
-}
\ No newline at end of file
+}
+
+export function sin(a) {
```
Comment (BenLorantfy) [inline math.js]: `a` is not a very well named parameter.  Did you consider a longer more expressive name?

Comment (ghost) [inline math.js]: Consider a better name for the parameter.

**Review by BenLorantfy — COMMENTED** [review]
Nice job me!

File: README.md [hunk 2]
```diff
@@ -10,3 +10,6 @@ https://education.github.com/git-cheat-sheet-education.pdf
 - Commit: A group of changes that is named and labelled with an ID
 - `git log`: Used to list the commits on the current branch
 - `git push`: Use this to publish your commits to the hosted repo
+- `git checkout -b`: Use this to create new branches
+- "Checked out branch": This means the current branch
+- `git checkout` / `git switch`: This means to switch branches
```
Comment (BenLorantfy) [inline README.md:15]: Consider adding more git commands here

**Review by ghost — COMMENTED** [review]
Consider a better parameter name.

File: README.md [hunk 3]
```diff
@@ -10,3 +10,6 @@ https://education.github.com/git-cheat-sheet-education.pdf
 - Commit: A group of changes that is named and labelled with an ID
 - `git log`: Used to list the commits on the current branch
 - `git push`: Use this to publish your commits to the hosted repo
+- `git checkout -b`: Use this to create new branches
```
Comment (thecanadianseahawk) [inline README.md:13]: Here's a comment

File: math.js (hunk 1 above)
Comment (BenLorantfy) [inline math.js:25]: this is crappy code

Comment (BenLorantfy) [inline math.js]:
```suggestion
export function sin(num) {
```

File: math.js [hunk 4]
```diff
@@ -20,4 +20,8 @@ export function sqrt(a) {
 
 export function log(a) {
     return Math.log(a);
-}
\ No newline at end of file
+}
+
+export function sin(num) {
+    return Math.sin(a);
```
Comment (BenLorantfy) [inline math.js]:
```suggestion
    return Math.sin(num);
```

File: README.md [hunk 5]
```diff
@@ -10,3 +10,6 @@ https://education.github.com/git-cheat-sheet-education.pdf
 - Commit: A group of changes that is named and labelled with an ID
 - `git log`: Used to list the commits on the current branch
```
Comment (Josers17) [inline README.md:11]: Comment

**Pull request merged**